lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,6941 @@
+{
```

Comment:
<p>Ako dobrá pomôcka pre začiatočníkov mi príde , že:</p><ul><li> <code>axis=0</code> sa tiež dá napísať ako <code>axis='index'</code> alebo <code>axis='rows'</code>.</li><li><code>axis=1</code> ako <code>axis='columns'</code>.</li></ul><p> </p><p>Dokumentácia tvrdí, že funguje pre všetky pandas funkcie s týmto argumentom, ale našiel som nejakú, kde to nešlo (už si ale nespomeniem).</p><p> </p><p>A keďže sme v pandas, tak <code>df.mean(axis='rows')</code> vracia priemer za jednotlivé stĺpce a <code>df.mean(axis='columns')</code> za jednotlivé riadky. Axis tu špecifikuje, po ktorej ose sa priemer počíta - logicky :-D.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Today I learnt ;-)</p>